### ISSUE TITLE
Send mosaics on testnet hangs

### ISSUE THREAD

**Issue by Dawn-Piper** [nemtech/symbol-cli#296]
1. Expected behavior and actual behavior.
Want to: Send mosaics from one account to another using the instructions [here.](https://nemtech.github.io/guides/transfer/sending-a-transfer-transaction.html) 
Get: 
`SUCESS Transaction announced`
`Processing`
and no transaction ever appears.

2. Steps to reproduce the problem.
Install nodejs@14.4.0 (latest)
Install symbol-cli@0.20.1

`symbol-cli profile create`
Select the network type: › TEST_NET  

Enter the Symbol node URL. (Example: http://localhost:3000): … http://api-01.us-east-1.testnet-0951-v1.symboldev.network:3000/  

Enter a profile name: … test  

Enter your wallet password: … ***********   

Do you want to set the account as the default profile? … yes

Send test xym from the [faucet](http://faucet-951-01.symboldev.network/?recipient=TCJCSY-N2DG5O-3TGBLS-QNELVL-C7CCNO-5BCVUA-PNND&amount=10) to this account.
Create another account using
`symbol-cli account generate`
The address of this account is the "recipient-address"
MOSAICID you can double check by 
`symbol-cli account info`
Finally:
`symbol-cli transaction transfer --recipient-address TBONKW-COWBZY-ZB2I5J-D3LSDB-QVBYHB-757VN3-SKPP --mosaics "MOSAICID::10000000" --message "This is a test message" --sync`

In summary, it's just a super basic transaction and it doesn't go through. It just hangs. `monitor all` on the recipient-address shows nothing. Similarly, all other transactions hang also.

3. Specifications like the version of the project, operating system, or hardware.
Ubuntu 18 (Running in an AWS instance in China)
Nodejs 14+ 
Symbol-Cli 20.1

**Comment by dgarcia360** [comment]
Please, execute this command ``npm install -g symbol-cli@latest`` to update the cli version. The ``transaction mosaic`` command received an update in the latest release v0.20.2. 

Also, I would recommend you to install the required [Node.js 12 LTS](https://nodejs.org/en/download/) (Long Term Support). The CLI has not been tested yet with newer Node.js versions yet.

**Comment by Dawn-Piper** [comment]
Thanks David,
node v12.18.0 & Symbol CLI v0.20.2 give the same result.

**Comment by Dawn-Piper** [comment]
I was not including the 6 zero
